src/PKSim.Core/Snapshots/Mappers/ExpressionContainerMapper.cs

```diff
@@ -48,10 +48,12 @@ public override async Task<ExpressionContainer> MapToSnapshot(MoleculeExpression
          if (transportedExpressionContainer.TransportDirection == TransportDirectionId.None)
             return null;
 
+         var isSurrogate = string.IsNullOrEmpty(expressionContainer.LogicalContainerName);
          var snapshot = await SnapshotFrom(expressionContainer, x =>
          {
-            x.Name = expressionContainer.LogicalContainerName;
-            x.CompartmentName = expressionContainer.CompartmentName;
+            x.Name = isSurrogate ? expressionContainer.Name : expressionContainer.LogicalContainerName;
```

Comment:
This was also not saved properly for global containers such as Blood Cells, VascularEndothelium etc..